app/controllers/application_controller.rb

```diff
@@ -43,4 +43,8 @@ def store_request_id
     RequestStore.store[:request_id] = request.uuid
     Raven.extra_context(request_id: request.uuid)
   end
+
+  def env
```

Comment:
I'm wondering what is the reason why Rails removed this `env` API in `ApplicationController` and wether it would be preferable to explicitly call `request.env` where it is needed?